### PR TITLE
Attempt to fix workflow

### PR DESCRIPTION
**Pull request by CristinaRO**

### CODE REVIEW

**Comment by erbridge** [comment]
@CristinaRO Is this ready for review?

**Comment by CristinaRO** [comment]
> @CristinaRO Is this ready for review?

I think so, yes.

File: script/fix [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+# script/fix: Run automated format fixers
+
+set -e
+
+cd "$(dirname "$0")/.."
+
+[ -z "$DEBUG" ] || set -x
+
+echo "==> Linting in fixer mode..."
```
Comment (dragon-dxw) [inline script/fix:11]: Should this line only appear if we're going to run the linters, i.e. inside the `if [ -n "$CI" ]` ?

**Review by dragon-dxw — APPROVED** [review]

**Comment by pezholio** [comment]
Is this still A Thing? If so, are you OK to merge @CristinaRO?

**Pull request merged**